### ISSUE TITLE
Better example of detached server required

### ISSUE THREAD

**Issue by paul-uz** [doapp-ryanp/dynamodb-local#51]
I'm struggling to get the detached server working reliably with Vitest.

The docs have the comment `// must be wrapped in async function` but do not offer a real example.

The Java process seems to launch, and some times, everything works OK, but then sometime the CreateTableCommand fails, or the process isn't stopped, meaning the next test to try and start the process might fail cos a table already exists etc.

Basically, please provide real world usage examples of how to use the detached server so tests can run properly, spinning up and stopping the java process as needed.
